config/gulp/release.js

```diff
@@ -0,0 +1,149 @@
+var gulp = require('gulp');
+var dutil = require('./doc-util');
+var spawn = require('cross-spawn');
+var del = require('del');
+var runSequence = require('run-sequence');
+var merge = require('merge-stream');
+
+var streamExtensions = [
+  'omnigraffle',
+  'eps',
+  'ai',
+  'sketch',
+];
+
+function createZipArchive (taskName, src, dest, done) {
+
+  var zip = spawn('zip', [
+    '--log-info',
+    '-r',
+    src,
+    dest,
+    '-x "*.DS_Store"',
+  ]);
+
+  zip.stdout.on('data', function (data) {
+
+    if (/[\w\d]+/.test(data)) {
+
+      dutil.logData(taskName, data);
+
+    }
+
+  });
+
+  zip.stderr.on('data', function (data) {
+
+    dutil.logError(taskName, data);
+
+  });
+
+  zip.on('error', function (error) {
+
+    dutil.logError(taskName, 'Failed to create a zip archive');
+
+    done(error);
+
+  });
+
+  zip.on('close', function (code) { if (0 === code) { done(); } });
+
+}
+
+function getAssetDirectory (ext) {
+  return dutil.dirName + '-' + ext;
+}
+
+gulp.task('clean:assets:directories', function () {
+
+  dutil.logMessage('clean:assets:directories', 'Deleting temporary assets directories.');
+
+  return del([
+    dutil.pkg.name + '*',
+    '!' + dutil.pkg.name + '*.zip',
+  ]);
+
+});
+
+gulp.task('clean:assets:archives', function () {
+
+  dutil.logMessage('clean:assets:archives', 'Deleting assets zip archives.');
+
+  return del([
+    dutil.pkg.name + '*.zip',
+  ]);
+
+});
+
+gulp.task('release:process', function () {
```

Comment:
Hey @rogeruiz, can you explain this task? I'm having a hard time understanding what it's doing.